code/_onclick/hud/screen_objects.dm

```diff
@@ -660,10 +660,10 @@
 			if(usr.locked_to && istype(usr.locked_to, /obj/structure/bed/chair/vehicle/adminbus))
 				var/obj/structure/bed/chair/vehicle/adminbus/A = usr.locked_to
 				A.Send_Home(usr)
-		if("Antag Madness!")
+		/*if("Antag Madness!")
```

Comment:
RIP best gamemode

Reply:
That's not a game mode, that was an adminbus button that made people into random antags, though that's where the idea came for the name of the mode I've been thinking about for when there's over 90% threat level (which isn't implemented by the way, but that's fine, admins can force it manually anyway)